File: Backend/Backend/Models/DTOs/UserRegisterRequestDTO/UserRegisterRequestDTO.cs
using System.Text.Json.Serialization;

namespace Backend.Models.DTOs.UserRegisterRequestDTO
{
    public class UserRegisterRequestDTO
    {
        public string Username { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public string Phone { get; set; }
        public List<string> Photos { get; set; }
        public string About { get; set; }
        public string Location { get; set; }

        [JsonConstructor]
        public UserRegisterRequestDTO()
        {
        }
    }
}

File: Mobile/app/src/main/java/com/example/mobile/DTOs/UserRegisterRequestDTO.kt
package com.example.mobile.DTOs

data class UserRegisterRequestDTO(
    public val username: UserToBeStoredDTO,
    public val password: String,
    public val age: String,
    public val email: String,
    public val name: String,
    public val phone: String,
    public val photos: ArrayList<String>,
    public val about: String,
    public val location: String,
)
